CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v7.7.1]
+
+### Fixed
+- Double Encoding of RQL values is now also supported in `OAuth1` clients
```

Comment:
(minor) Might it be interesting to mention we reimplemented the oAuth1 signature generation for this? If we made a mistake and receive oAuth signature errors, maybe users would make the connection that upgrading to this version is to blame